stretch.js

```diff
@@ -104,5 +108,17 @@ Output:
 */
 
 export function countByCategory(arr) {
-    return {};
+    const categories = {};
```

Comment:
Nice, a reduce will also do this in less lines of code...but I'm impressed with this. Nice work!